### DISCUSSION TITLE
如果世上有後悔藥，你會如何選擇？

### DISCUSSION THREAD

**Discussion by Miinteaa** [1101-GS4719-C/Python-Community-GS4719-C#1399] [General]
在人生中無時無刻都面臨著選擇，小至思考早午晚餐要吃什麼，大到決定自己的人生規劃（如出國留學等等），但人生不像電腦遊戲一般，選錯了選項、走入錯誤的結局，還能夠讀取存檔重新再來，而是只有一次的機會，很可能就決定了接下來的劇情發展。

**後悔是人們常有的、一種回溯性的情緒**，口中總說著：「要是當初⋯⋯就會⋯⋯」來表達自己對於現狀的不滿，設想自己的人生從那一刻起原本有著更好的可能。但這種情緒在隱隱之中限制了我們，過度執著於過去的選擇導致人們無法向前邁進，桎梏於原地。

有時人們甚至會主動選擇一直停留在後悔裡，即便當他們有機會重新作出一些不同的選擇時，他們也並不會這樣做，而是繼續停留在後悔中。**他們更難面對的是，那種更好的可能，在真正做出嘗試之後破碎了。**

### 如何科學的處理後悔的情緒呢？也許你可以嘗試以下幾個方法：
**1. 意識到你有沒有在從後悔中，獲得不該從中尋找的情緒價值感？** 
比如，你是否一直在後悔當初沒有做好一些事，導致你失去了愛人，其實是不願意面對「對方就是沒那麼愛你」的事實？

**2. 降低後悔的感受：**
如果後悔這種情緒經常困擾到你，你需要解決的問題可能並不是提高決策能力。而是降低自己對於社會比較的需求。
研究顯示(VanDijk,E.,&Zeelenberg,M,2005)，更傾向於與他人比較、並從中獲得價值感的人，更容易感到後悔。因為我們並不總是能對比出選項之間的差距。同一個研究顯示，當我們不知道自己錯過了什麼的時候，我們的大腦是不太會感到後悔的。而當所錯過的東西不具體、很難評估時，我們感受到的後悔程度也會更輕。但傾向於社會比較的人，會把他人的選擇結果，直接當作自己所錯過的加以比較。

**3. 其實你的「do-over」（再來一次的機會）不在當初，就在你回頭看的此時。** 
犯錯不但不是意外、不是不該發生的事，而是必然、是歡迎發生的事。如果我們能將每一個錯誤，都視為一個成長的契機，那麼我們也能將後悔的情緒淡化。

### 我的看法：
於我個人而言，人生至此好像並沒有什麼後悔的事物長期困擾著我，雖然有一些小煩惱，但回頭來看這些決定就像拼圖一樣一片片拼湊出現在這個完整的自己，沒有這些決定，以及決定後帶給我的經歷，現在的我可能就會是一個完全不同的人。也許在後悔之前，我們能夠想的是這個決定帶給了我們什麼又或是我們能改變什麼，而非專注於那些不曾擁有或虛無的幻想。

你也有後悔過的事物嗎？若是能夠回到當初你會怎麼選擇呢？

**Comment by KozakHou** [comment]
我看了最新的閃電俠電影預告，裡面有句話好有意思。

不要去改變過去，因為他造就現在的你。

這段話我久久無法自已，顛覆了我對於要是能重來這句話的意義。

這種事實對本人來說意義重大，相信對這個世界也是有一定意義的。在人類的歷史中，我們總是盡了一切努力想搞懂要是能重來。儘管如此，別人往往卻不這麼想。

所謂要是能重來，關鍵是要是能重來需要如何解讀。

**Comment by alicia0725** [comment]
我覺得很多後悔的事，只是因為在我的想象中，當初沒被我所選擇的會很成功，但實際上這只存在我的想象中。因此我認為既然沒辦法蟲頭來過，我應該要更專注在我現在的選擇上，而不是在怨天尤人的後悔

**Comment by pigeon9975** [comment]
高中時，我很後悔會考時我所做的決定，我總是在想如果當初降一個分數填，高中的我會不會過得比較快樂？是不是就不用遇到那些我不喜歡的人？是不是在學業上就不會碰到那麼多挫折？但現在回頭想想，或許是因為高中這樣的環境才能造就出現在的自己，也或許是因為那樣的經歷才會讓自己更珍惜現在所擁有的一切。沒有甚麼後不後悔的，就算後悔也無法回頭，向前走，把失意的那段時光當作養分，才是最從明、最成熟的選擇。

**Comment by zephyrxin01** [comment]
我問了一下chatGPT如何避免做出後悔的選擇，他給出了幾點滿務實的建議:
1.多閱讀：閱讀是擴展知識和觀點的重要方式。通過閱讀相關領域的書籍、文章和研究報告，你可以學習到更多關於該主題的信息和經驗。這將使你更加了解不同的觀點和可能的選擇，從而做出更全面的決策。
2.聆聽前人的經驗分享：前人的經驗可以提供寶貴的教訓和指導。聆聽他們的故事和分享，可以從他們的成功和失敗中獲得啟示。這可以通過參加講座、工作坊、行業研討會或與經驗豐富的人交流來實現。
3.建立良好的支持網絡：與他人交流和討論你的決策可以提供新的觀點和反饋。找到可靠的朋友、導師或專家，向他們尋求意見和建議。他們可以從不同的角度看待問題，幫助你更好地理解和評估你的選擇。
4.深入研究和分析：在做出重要決策之前，進行詳細的研究和分析是必要的。收集和整理相關的數據和信息，使用分析工具來評估不同選項的風險和潛在結果。這將為你提供更可靠的依據，幫助你做出更明智的決策。
5.利用傾聽內在直覺：除了依靠知識和分析，傾聽你的內在直覺也是重要的。有時候，直覺可能是你潛意識中對某個選項的感覺或預感，值得關注和考慮。當你在經歷一個決策困境時，學會信任和聆聽你的直覺。

對我來說，聽前人的經驗分享來避免做出後悔的選擇的確很有說服力，但實在很難避免自己不見棺材不掉淚阿...

**Comment by Chiuhsianglin** [comment]
活到現在，我有很多後悔的事情，但有時候仔細想想，但後來想想或許那些讓我們後悔的事情也可以幫我們成長，從錯誤中學習！

**Comment by ALAN30359704** [comment]
我覺得後悔並沒有錯，但是要從後悔中去反省，並且把握當下在這個後悔之中學到的教訓以後不要再犯，這才是從經驗中學習而得到成果，但我認為相對重要的是不要沉浸在這個後悔的情緒中過久要恢復精神繼續向前！

**Comment by zephyrxin01** [comment]
我覺得後悔藥的存在很麻煩，感覺一不小心就會有人上癮。畢竟人生不是由單一事件組成，有時候事情是並行存在的，一個決定的改變可能大大影響了整個事件的脈絡，因此並不是調整其中一個決定就能讓事情的發展往自己預期的方向走。

**Comment by sjh501** [comment]
對我來說後悔都是暫時的一個情緒而已，畢竟事情發生了不可逆，因為自己或者別人的選擇所造成的不愉快並不能直接的解決事情。我覺得不是不能後悔，而是在後悔之後做了什麼事情，因為這個事情學習到什麼、了解到什麼，好像才比較有幫助。但如果一直執著在後悔的事情很有可能會開始懷疑自我，那時後悔藥說不定會有幫助。

**Comment by alstinlin** [comment]
雖然碰到過許多讓自己後悔的事，但過一段時間想起來也比較不會在意了，再說做了決定本來就會失去一些東西，但相對的也會有其他收穫，所以比起後悔，不如繼續前進，說不定結果會比預期的來的好

**Comment by pudding0803** [comment]
我認為在一件事情上不應該後悔太久，重要的是知道自己做出該決定的原因與錯誤，接著檢討並在未來加以避免或改進。人畢竟沒辦法隨意體驗各種選擇帶來的後果，因此除非太明顯是錯誤的，否則後悔太久並沒有太大的意義。

假設現在是在時間線 A，當初做了另一個選擇就會到時間線 B。加設 A 發生了一些不好的事，我們並沒有辦法得知 B 是否一定比 A 好，或是在未來也一定是 B 比較好。舉個奇怪浮誇的例子🤣，假設 P、Q 去買樂透，P 畫完號碼覺得不切實際，因此沒有買就走了，而 Q 買了。過了幾天他們發現那些號碼居然全中，如果他當初買下去就會中頭獎，P 可能就會說：「早知道就買了！」。但幾個禮拜後，因為有人忌妒 Q 的財產，就把 Q 害得生不如死，雖然犯人被逮到，但 Q 失去的一切都已經不再復返，Q 可能就會說：「早知道就不買了！」

我們永遠沒辦法知道哪個選擇才是最正確的，因為選擇就像河流一樣，會有非常非常多的分支，或許是你過去做了某個錯誤的選擇，才讓你現在具備下一次成功的條件。因此如果你會後悔，審視後悔的原因並加以改進就好了！或許多多思考也可以幫助你更了解自己的想法！

**Comment by 1104030360** [comment]
我認為如果有後悔藥，我不會吃下後悔藥，因為就是有過去的種種事情才會造就現在優秀的我們，如果我們有了後悔藥，我們的人生可能會就此改變，變成不能承擔壓力的人（因為都會想著吃下後悔藥就好了ㄚ），有了後悔才有成長～

**Comment by JessyCheng2003** [comment]
我覺得人會後悔，多數是認為另一個選擇會更好，但我覺得選擇本身是沒有好壞之分，而是要對自己的選擇負責。

**Comment by lyien1021** [comment]
本來我是覺得沒什麼事值得後悔，因為隨著時間推移，再度回首時常常會發現當初後悔的事反而不一定是件壞事，所有經歷都造就了現在的你。不過自從我這禮拜一打球扭到腳之後我就改觀了，要是有後悔藥我一定會好好看路不要跌倒。

**Comment by Brownnnn78** [comment]
後悔藥對我自己來說沒有這個慾望，因為每個時刻都在造就現在的我，然而我會為了別人而吃，很想，後悔自己錯過了某些人、後悔某些不該錯過的關心等等，雖然這些不會影響我自己，但或許能拯救其他人。

**Comment by Elaine2899** [comment]
小時候會有很多後悔的事情，想回到過去改變的事情，但現在的我覺得正是那些事情成就現在的我，讓我可以更成熟更有能力做不讓自己後悔的決定。

**Comment by Evening218** [comment]
我覺得如果從極度理性的角度來看,後悔這件事應該要分兩種,如果是自己一時疏忽,那就要反省,告訴自己下次注意,如果是自己無法掌控的因素,那就別後悔了因為真的沒用
